src/Explorer/Tree/Collection.ts

```diff
@@ -1052,6 +1045,41 @@ export default class Collection implements ViewModels.Collection {
     return deferred.promise;
   };
 
+  public async getPendingThroughputSplitNotification(): Promise<DataModels.Notification> {
```

Comment:
Is this still used? I thought this data came down via a header in the offers call? I could be wrong here.

Reply:
From the code it looks like it's still being used, but I'm not sure if it's actually needed in any scenario.  This function fetches notification from the portal by calling `fetchPortalNotifications` so it's different from the `offerReplacePending` header.

I think we should keep it for now just to be safe and I can investigate whether we actually need this or not.